modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -13,183 +15,96 @@ $ oc netobserv [<command>] [<feature_option>] [<command_options>] <1>
 ----
 $ oc netobserv flows [<feature_option>] [<command_options>]
 ----
-
-.Flow capture filter options
+[cols="1,1,1",options="header"]
 |===
-|Option| Description| Possible values| Mandatory| Default
-
-| `--enable_filter`
-| Enable flow filter.
-| `true`, `false` 	
-| Yes 
-| `false`
-
-| `--action`
-| Action to apply on the flow.
-| `Accept`, `Reject`
-| Yes
-| `Accept`
-
-| `--cidr`
-| CIDR to match on the flow. 
-| `1.1.1.0/24`, `1::100/64`, or `0.0.0.0/0`
-| Yes
-| `0.0.0.0/0`
-
-| `--protocol`
-| Protocol to match on the flow
-| `TCP`, `UDP`, `SCTP`, `ICMP`, or `ICMPv6` 	
-| No
-| -
-
-| `--direction`
-| Direction to match on the flow
-| `Ingress`, `Egress`
-| No
-| -
-
-| `--dport`
-| Destination port to match on the flow.
-| `80`, `443`, or `49051`
-| no
-| -
-
-| `--sport`
-| Source port to match on the flow.
-| `80`, `443`, or `49051`
-| No
-| -
-
-| `--port`
-| Port to match on the flow.
-| `80`, `443`, or `49051`
-| No 
-| -
-
-| `--sport_range`
-| Source port range to match on the flow.
-| `80-100` or `443-445`
-| No 	
-| -
-
-| `--dport_range` 	
-| Destination port range to match on the flow.
-| `80-100`
-| No
-| -
-
-| `--port_range`
-| Port range to match on the flow.
-| `80-100` or `443-445`
-| No
-| -
-
-| `--icmp_type`
-| ICMP type to match on the flow.
-| `8` or `13`
-| No
-| -
-
-| `--icmp_code`
-| ICMP code to match on the flow.
-| `0` or `1`
-| No
-| - 	
-
-| `--peer_ip`
-| Peer IP to match on the flow.
-| `1.1.1.1` or `1::1`
-| No 	
-| -
+| Option | Description | Default
+|`--enable_pktdrop`|  enable packet drop                  | false
+|`--enable_dns`|      enable DNS tracking                 | false
+|`--enable_rtt`|      enable RTT tracking                 | false
+|`--enable_filter`|   enable flow filter                  | false
+|`--log-level`|       components logs                     | info
+|`--max-time`|        maximum capture time                | 5m
+|`--max-bytes`|       maximum capture bytes               | 50000000 = 50MB
+|`--copy`|            copy the output files locally       | prompt
+|`--direction`|       flow filter direction               | n/a
+|`--cidr`|            flow filter CIDR                    | 0.0.0.0/0
+|`--protocol`|        flow filter protocol                | n/a
+|`--sport`|           flow filter source port             | n/a
+|`--dport`|           flow filter destination port        | n/a
+|`--port`|            flow filter port                    | n/a
+|`--sport_range`|     flow filter source port range       | n/a
+|`--dport_range`|     flow filter destination port range  | n/a
+|`--port_range`|      flow filter port range              | n/a
+|`--tcp_flags`|       flow filter TCP flags               | n/a
+|`--icmp_type`|       ICMP type                           | n/a
+|`--icmp_code`|       ICMP code                           | n/a
+|`--peer_ip`|         peer IP                             | n/a
+|`--action`|          flow filter action                  | Accept
```

Comment:
all flow filter related options can be one below the other. so action has to be moved up a couple of rows